packages/react-component-library/src/components/DatePicker/partials/StyledCalendarNavigation.tsx

```diff
@@ -0,0 +1,114 @@
+import styled, { css } from 'styled-components'
+import { color, fontSize, spacing } from '@royalnavy/design-tokens'
+
+import { Button } from '../../Button'
+
+export const StyledNMYContainer = styled.div`
```

Comment:
I didn't find it very obvious what the `NMY` in `StyledNMYContainer` means (also it's hard to read).

Is there a better name that can be used?

Reply:
Looks like this is redundant - i've removed it and fixed the styles.